backend/src/Squidex/Areas/Api/Controllers/Contents/Models/ContentDto.cs

```diff
@@ -179,7 +179,10 @@ private ContentDto CreateLinksAsync(IEnrichedContentEntity content, Resources re
 
             if (content.CanUpdate && resources.CanUpdateContent(schema))
             {
-                AddPutLink("update", resources.Url<ContentsController>(x => nameof(x.PutContent), values));
+                if (resources.CanUpdateContentOwn(schema))
```

Comment:
This is not necessary, we only have to update the permissions in the resources file and keep everything as it is.